simulation/constants.py

```diff
@@ -128,3 +128,25 @@ class EventTrigger(enum.Enum):
     HeatingSystem.HEAT_PUMP_AIR_SOURCE: 2.0,
     HeatingSystem.HEAT_PUMP_GROUND_SOURCE: 2.0,
 }
+
+
+class PropertySize(enum.Enum):
+    SMALL = 0
+    MEDIUM = 1
+    LARGE = 2
+
+
+# Source: https://www.ovoenergy.com/guides/energy-guides/how-much-heating-energy-do-you-use
+# Assume figure of 133kWh/m2a a reflects an average heating system Coefficient of Performance of 0.92 (gas boiler)
+# 133 * 0.92 = 122kWh/m2a
+HEATING_KWH_PER_SQM_ANNUAL = 122
+
+COEFFICIENT_OF_PERFORMANCE: Dict[HeatingSystem, float] = {
```

Comment:
I think this perhaps "overloads" the term "Coefficient of Performance" to include the "efficiency" of the boilers as a "coefficient of performance". I really struggled to come up with a name for this term last time, I'm happy to define it as this, or instead to something like `FUEL_KWH_TO_HEAT_KWH`